test/uk/gov/hmrc/helptosavefrontend/controllers/RegisterControllerSpec.scala

```diff
@@ -617,9 +617,9 @@ class RegisterControllerSpec
 
         val result = getAccountCreatedPage()
         status(result) shouldBe OK
-        contentAsString(result) should include("Account created")
+        contentAsString(result) should include("Continue")
```

Comment:
I am not sure this is the best check that the correct page has rendered, this change only asserts a continue button exists somewhere on the page - whereas the previous check was for the h1  content - if we match that instead then this assures us the correct page renders.